### DISCUSSION TITLE
Combining Durables

### DISCUSSION THREAD

**Discussion by Tangsoodo2** [postmanlabs/postman-flows#279] [Q&A] [answered]
Just learning about postman flows (great idea and could help QA understand different use cases!),  and trying to accomplish a specific use case.... 

I have multiple requests filtering data into Durables.  I'm looking to combine durables into a variable that goes into a request.  Consider the example below

Durable1 = 123
Durable 2 = 333

What I'm trying to do is combine the durables into 
Durable3/Variable = 123,333

I have a request where I'm trying to inject Durable3 into,  something like 

<base>
<example>{{Durable3}}</example>
</base>

Screenshot below of my current flow

![image](https://user-images.githubusercontent.com/6608023/157994600-9641dbc1-3d55-4a1c-b023-a8617357099a.png)

Any help would be much appreciated!

**Comment by saswatds** [comment, marked as answer]
Hey @Tangsoodo2. If Durable 1 is a string and Durable 2 is a string. You can do something like this with assign value to variables. 
![Screenshot 2022-03-15 at 11 22 07 AM](https://user-images.githubusercontent.com/8796447/158315607-b069d41a-49ca-4883-a94f-b273a36ae1d8.png)

But, I do see a lot of potential for improvement in this workflow. I will keep this thread posted when we implement a simpler way.

**Reply by Tangsoodo2** [reply]
Thanks for the reply @saswatds. That will solve my problem. I wasn't considering changing the request to allow both variables (instead of making the workflow concatenate the two variables).  Knew there was something easy I was missing. 

Being able to toString and concatenate variables might be a helpful enhancement in the future.

**Reply by saswatds** [reply]
The `to string` or `as string` to concatenate was exactly what I has in mind for the next round of improvement.
